projects/igniteui-angular/src/lib/grids/grid-navigation.service.ts

```diff
@@ -553,6 +555,15 @@ export class IgxGridNavigationService {
             });
         this.horizontalScroll(rowIndex).scrollTo(unpinnedIndex);
     }
+
+    private focusGridWithoutScrolling() {
+        if(isIE() && (<IgxHierarchicalGridComponent>this.grid).parent) {
+            (<IgxHierarchicalGridComponent>this.grid).parent.nativeElement.focus();
```

Comment:
This may not work when navigating in nested children. Use the hierarchical grid’s rootGrid  prop instead to get the root grid.